docs/nextflow_schema/nextflow_schema_specification.md

```diff
@@ -367,18 +373,9 @@ Example usage is as follows:
 }
 ```
 
-!!! note
-
-    If `exists` is set to `false`, this validation is ignored. Does not check if the path exists.
-
-!!! note
-
-    If the parameter is set to `null`, `false` or an empty string, this validation is ignored. It does not check if the path exists.
```

Comment:
Did we remove this feature?

Reply:
I expanded `exists` to now also check if the file does not exist when it's set to `false` all other inputs will ignore this